Updated Store Library and Examples

jvelilla opened the pull request:
* Get rid off obsolete calls.
* Added encoding library.
* Clean code, removed unused variables.
* Updated obsolete messages without a deadline date.

jvelilla committed:
Updated Store Library and Examples
* Get rid off obsolete calls.
* Added encoding library.
* Clean code, removed unused variables.
* Updated obsolete messages without a deadline date.

manuseiffel commented on Src/examples/store/insert/inserter.e:
```diff
@@ -85,7 +85,7 @@ feature {NONE} -- Initialization
 						-- The attempt to insert a new object
 						-- failed
-					io.putstring (session_control.error_message)
+					(create {LOCALIZED_PRINTER}).localized_print (session_control.error_message_32)
```
It might be best to simply inherit from LOCALIZED_PRINTER instead.

manuseiffel commented on Src/examples/store/st_proc_numeric/runner.e:
```diff
@@ -115,14 +116,14 @@ feature {NONE}
+				if l_proc.text_32 /= Void then
 					io.putstring ("Stored procedure text: ")
-					io.putstring (l_proc.text)
+					io.putstring (l_utf.string_32_to_utf_8_string_8 (l_proc.text_32))
```
Shouldn't you use LOCALIZED_PRINTER here?

jvelilla replied:
Yes, I will fix it.

manuseiffel reviewed (approved):
I would change the obsolete date to 2017-11-30 to give time to our users to update when the code is released to them.

jvelilla committed:
Updated code to inherit from LOCALIZED_PRINTER
Updated obsolete messages deadline date to use 2017-11-30 instead of 2017-05-31

manuseiffel reviewed (approved):
LGTM

manuseiffel commented:
Committed in svn rev#100428 and rev#1000429
